### PR TITLE
Fe4341: ListingBody component exported unconnected as ListingBodyComponent

### PR DESCRIPTION
**Pull request by ajmalwali7**
#4341 issue solved as 'ListingBody' exported with unconnected and as 'ListingBodyComponent'

### CODE REVIEW

**Comment by stevepiercy** [comment]
@ajmalwali7 please see https://6.docs.plone.org/volto/developer-guidelines/contributing.html#create-a-pull-request

File: news/4341.internal [hunk 1]
```diff
@@ -0,0 +1 @@
+Fixed the issue #4341 @ajmalwali7
```
Comment (stevepiercy) [inline news/4341.internal:1]: A news item should include what you changed, as well as referring to the issue number. This helps people who read the change/history log. See examples: https://6.docs.plone.org/volto/release-notes/index.html

**Review by stevepiercy — CHANGES_REQUESTED** [review]

File: news/4341.internal (hunk 1 above)
Comment (ajmalwali7) [reply news/4341.internal]: Done sir, You can check it out.

**Comment by wesleybl** [comment]
@ajmalwali7 I suggest you close this PR and open a new one, using a branch of the repository itself, instead of its fork. Volto has an issue where it does not test PRs created from forks.

**Comment by ajmalwali7** [comment]
@wesleybl thanks man, I will do that. I am new to contributions so that's why I don't know many rules. Btw thanks

**Comment by stevepiercy** [comment]
@wesleybl @ajmalwali7 this is a chicken-or-egg issue: only people who have sent in a signed Contributor Agreement and have been approved and added to the GitHub @plone/developers Team may open a branch in this repository.

We could open up CI/CD to run on external branches. That's up to @plone/volto-team to discuss and decide.

**Pull request closed without merging**